package.json

```diff
@@ -73,12 +73,12 @@
     "@storybook/addons": "^6.1.17",
     "@storybook/react": "^6.1.17",
     "@storybook/source-loader": "^6.1.17",
-    "@testing-library/dom": "^6.1.0",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.1.3",
+    "@testing-library/dom": "^7.29.4",
```

Comment:
Would it be possible to split out the updates for `@testing-library` from the updates for `react` into separate pull requests?

Reply:
Yes, I created a separate PR for the testing library https://github.com/grommet/grommet/pull/4999